### DISCUSSION TITLE
TRT 9.1 的奇怪崩溃

### DISCUSSION THREAD

**Discussion by hooke007** [AmusementClub/vs-mlrt#65] [General]
触发条件很奇怪。在mpv里运行vs滤镜，seek跳转播放的时候很大几率程序崩溃，查过mpv的日志没有价值的信息，这种模式下也不知道如何输出vs的日志。
回滚到[trt 8.6](https://github.com/AmusementClub/vs-mlrt/releases/tag/v14.test) 一切正常（替换 vstrt.dll 和 vsmlrt-cuda文件夹），跳转不再崩溃。

**Comment by WolframRhodium** [comment]
Thanks. There's also been some similar reports on the SVP forums recently. I don't have much of an idea at this time. It's pretty strange that 8.6 doesn't have the problem though.

If the problem is a result of interaction of CUDA and OpenGL/graphics interfaces in NVIDIA's driver or new features of mpv, that's out of the control of me as a developer. (and VS scripting usually don't use graphics)

(also TensorRT 9.1 is not officially released for models other than LLMs. Probably its developer could fixed the problem later. DPIR does not compile on TRT 9.0.0, for example)

**Comment by WolframRhodium** [comment]
Could you please check whether the latest build fix this problem?

I can reproduce it and I hope https://github.com/AmusementClub/vs-mlrt/commit/2b7c1cbbf0dff3c2fc8f8ebad17bd7f6a9167c62 solve it (although imperfectly).

**Reply by hooke007** [reply]
Yes, fixed.

**Reply by WolframRhodium** [reply]
Thanks!
